### ISSUE TITLE
Console.info statements in webpack config file causes resolve error

### ISSUE THREAD

**Issue by tinyearth** [steelbrain/linter-ui-default#542]
### Environment:
- linter-ui-default 1.7.1
- Atom 1.37
- webpack 4
- Mac 10.13.6

### Example webpack config file: 
```
const webpack = require('webpack');
const path = require('path');

module.exports = () => {
    const configToUse = {
        cache: true,
    };

    // eslint-disable-next-line
    console.info('This is breaking linter-ui-default.');
    return configToUse;
};
```

### Result: 
All import statements in a repository's files are flagged as "Unable to resolve path to module". Below is a screenshot of the resolve errors from Atom's bottom dock.

<img width="761" alt="bottom-dock-resolve-errors" src="https://user-images.githubusercontent.com/2474510/58582101-da465300-8205-11e9-9933-cdb2e15f481b.png">

**Comment by Arcanemagus** [comment]
If this was an issue with a provider, it would need to be filed on [linter-eslint](https://github.com/AtomLinter/linter-eslint), however that looks to be a problem either with [eslint-plugin-import](https://github.com/benmosher/eslint-plugin-import), or possibly your configuration of it.

**Comment by tinyearth** [comment]
@Arcanemagus can you help pinpoint where this issue should go to? 

I'm thinking it's [linter-eslint](https://github.com/AtomLinter/) as I'm not quite understanding how [eslint-plugin-import](https://github.com/benmosher/eslint-plugin-import) relates. The only other Atom linter packages I have installed (which I've made no configurations to) are:
- linter 2.3.0
- linter-eslint 8.5.5

**Comment by Arcanemagus** [comment]
All of the rules shown in your screenshot are coming from `eslint-plugin-import`, and it's a notoriously buggy plugin in my experience. If you want to file an issue on `linter-eslint` **with** an example repository that shows the issue we can track it down there.

**Comment by tinyearth** [comment]
Gotcha, thanks. 

Unfortunately, I don't have a readily available public repo I'm able to provide at the moment. I've created an [issue in eslint-plugin-import](https://github.com/benmosher/eslint-plugin-import/issues/1373) for now. Hopefully they'll be able to investigate with what I've provided.
